iiitdmj/package.json

```diff
@@ -1,16 +1,18 @@
 {
-  "name": "iiitdmj",
+  "name": "college-website",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "node-sass": "^6.0.1",
+    "@reduxjs/toolkit": "^1.6.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.5",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2"
+    "sass": "^1.42.1",
+    "sass-loader": "^12.1.0"
```

Comment:
I guess node-sass is better in this condition, as it has better support for node/react based app, still we need to have a proper discussion on which module to use.